### PR TITLE
added comment

### PR DESCRIPTION
**Pull request by Grizvok**
First pull request

### CODE REVIEW

File: dashboard.js [hunk 1]
```diff
@@ -168,4 +168,6 @@ knex('clients.thigh').insert({user_id: user, thigh: measurement, date: new Date(
     res.end();
 });
 
+//this is where endpoints for updating client information will soon go
```
Comment (mattjs) [inline dashboard.js:171]: dude this comment sux

**Review by mattjs — CHANGES_REQUESTED** [review]

File: dashboard.js (hunk 1 above)
Comment (mattjs) [reply dashboard.js:171]: just kidding its fine, but now update the comment some more

Comment (Grizvok) [reply dashboard.js:171]: get rekt bro

**Review by mattjs — APPROVED** [review]
LGTM

**Pull request merged**